Mathlib/ModelTheory/Equivalence.lean

```diff
@@ -194,6 +194,20 @@ protected theorem imp {φ ψ φ' ψ' : L.BoundedFormula α n} (h : φ ⇔[T] ψ)
     BoundedFormula.realize_imp]
   exact fun M v xs => imp_congr h.realize_bd_iff h'.realize_bd_iff
 
+protected theorem sup {φ ψ φ' ψ' : L.BoundedFormula α n}
```

Comment:
```suggestion
protected theorem sup_congr {φ ψ φ' ψ' : L.BoundedFormula α n}
```
Also maybe you want to depend on @awainverse's PR which also proves this lemma

Reply:
This theorem is not used in this PR, so I can actually just remove it.